README.md

```diff
@@ -1,17 +1,48 @@
+[![This project uses GitHub Actions for continuous integration.](https://github.com/ember-learn/guidemaker-ember-template/workflows/CI/badge.svg)](https://github.com/ember-learn/guidemaker-ember-template/actions?query=workflow%3ACI)
+[![This project uses Percy.io for visual regression testing.](https://percy.io/static/images/percy-badge.svg)](https://percy.io/Ember/guidemaker-ember-template)
+
 # guidemaker-ember-template
 
-[Short description of the addon.]
+This addon provides the general layout and some styles that are used for documentation websites such as [guides.emberjs.com](https://guides.emberjs.com/release/), and [cli.emberjs.com](https://cli.emberjs.com/release/).
 
-## Compatibility
+Its responsibilities include
+- CSS for the sidebar
+- HTML markup specifics (i.e. the classes that go on various divs in the text of the guide itself)
+
+## Related apps
+
+Most aspects of the Guides are styled via [ember-styleguide](https://github.com/ember-learn/ember-styleguide), not this repository.
 
+[guidemaker](https://github.com/empress/guidemaker) is what assembles guides content into queryable records.
+
+[guides-source](https://github.com/ember-learn/guides-source) has the learning content itself.
+
+## Compatibility
 - Ember.js v4.12 or above
 - Embroider or ember-auto-import v2
 
 ## Installation
-
-```
-ember install guidemaker-ember-template
-```
+- `git clone <repository-url>`
+- `cd guidemaker-ember-template`
+- `pnpm install`
+
+## Linting
+- `pnpm lint`
+- `pnpm lint:fix`
+
+## Building the addon
+- `cd guidemaker-ember-template`
+- `pnpm build`
+
+## Running tests
+- `cd test-app`
+- `pnpm test` – Runs the test suite on the current Ember version
+- `pnpm test:watch` – Runs the test suite in "watch mode"
+
+## Running the test application
+- `cd test-app`
+- `pnpm start`
+- Visit the test application at [http://localhost:4200](http://localhost:4200).
```

Comment:
this stuff should be left as it is, the information that you're talking about is already in the contributing guide https://github.com/ember-learn/guidemaker-ember-template/blob/main/CONTRIBUTING.md#installation and this is actually a package that can be installed (not that many people will 😂 )